apps/addons/cron.py

```diff
@@ -453,9 +453,7 @@ def reindex_addons(index=None, addon_type=None):
     from . import tasks
     ids = (Addon.objects.values_list('id', flat=True)
```

Comment:
Or maybe it does? I'm lost now. You say it needs at least one run that doesn't exclude them, but then you need to include them, right? Maybe you even need `Addon.unfiltered....`?